### ISSUE TITLE
Media: file details screen labeled as "space", file detail titles are not read if they have content, copy button not labeled as a button

### ISSUE THREAD

**Issue by designsimply** [wordpress-mobile/WordPress-Android#10938]
Steps to reproduce:

1. Go to Settings > Accessibility > TalkBack > "Use service" or use the [Volume key shortcut](https://support.google.com/accessibility/android/answer/6006966?hl=en).
1. In the WordPress app, go to My Site > Media.
1. Open the global context menu by swiping down then right and double tap "Read from top".

**Result:** ([1m8s](https://cloudup.com/ccJCVwY6KzV))

* On the main media grid, the instruction read aloud for a thumbnail says "double tap to activate, double tap and hold to long-press" but double tapping and holding to long-press does nothing.
* The first focusable item on screen is read as "space".
* Field titles under File Details are not read aloud if they contain content.
* "Copy" button under URL is not labeled as a button.

<img src="https://user-images.githubusercontent.com/1119271/70568040-4cec9f80-1b54-11ea-9bff-e212afe99b39.png" width="270" alt="media-details-screen">
<sup>Tested with WPAndroid 13.8-rc-1 on Pixel 3 Android 10.</sup>

<p style="clear:both;"><sub>/hat tip @mchowning for the report, internal reference: p77Llu-c0a-p2</sub></p>

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
